service/src/main/java/com/fecklessweasel/service/objectmodel/University.java

```diff
@@ -249,4 +251,32 @@ public static University lookup(Connection connection, String longName)
             throw new ServiceException(ServiceStatus.DATABASE_ERROR, ex);
         }
     }
+
+    /**
+     *@author Hayden Schmackpfeffer
+     *@param connection The SQL Connection
+     *@return the list of ALL universities in the database
+     */
+    public static ArrayList<University> getAll(Connection connection) throws ServiceException{
```

Comment:
ArrayList<University> should be abstract List<University> or better yet Iterable<List> type to maintain flexibility of implementation.